.gitignore

```diff
@@ -1,4 +1,6 @@
 *.swp
+*__pycache__
+*.pip
 .env
 chromadb-data
```

Comment:
These three are no longer needed, now that you use volumes,

Reply:
we still need it, because we run this command: `pip -q install --cache-dir=.pip -r requirements.txt`

Reply:
No, sorry I was not clear. You no longer need the lines for `*-data` in the `.gitignore`. The Python and Pip ones are absolutely fine.

Reply:
done